.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: node_js
+script: echo "It's alright!"
```

Comment:
Can someone explain why Github / Git shows these lines as new and does not recognize it as already given in the current state of the develop-branch? And does it mean that I should not use rebase to get a feature-branch "updated"?